src/app/app.component.html

```diff
@@ -1,5 +1,13 @@
 <app-header></app-header>
 <div class="wrapper">
-  <router-outlet></router-outlet>
+  <div class="content-wrapper">
+    <router-outlet></router-outlet>
+    <app-main-search-box></app-main-search-box>
```

Comment:
Please contact @LilithPrimary and discuss refactoring for this block + plan the merge process of our features

Reply:
done